resources/js/Pages/Dashboard.tsx

```diff
@@ -36,6 +37,120 @@ export default function Dashboard() {
                             View my shared cards
                         </button>
                     </div>
+                    {/* this is all dummy content for testing and should be deleted eventually */}
+                    <div className="flex w-full flex-wrap med:mx-[-2]">
+                        <div className="mb-2 w-full px-2 md:w-1/2 lg:w-1/4">
+                            <p className="text-center">Double sided card</p>
+                            <MTGCard
+                                imgSrc="https://cards.scryfall.io/large/front/1/8/18a2bdc8-b705-4eb5-b3a5-ff2e2ab8f312.jpg?1626663778"
+                                title="Wandering Archaic"
+                                cardSuperType="Creature"
+                                cardType='Avatar'
```

Comment:
@dazu1990 should be double quote as per lint rules, suprised prettier didn't catch this.

Reply:
fixing manually - let's pair tonight to see why prettier isn't catching it